Source/SwiftLintFrameworkTests/ASTRuleTests.swift

```diff
@@ -148,6 +148,10 @@ class ASTRuleTests: XCTestCase {
             ruleDescription: FunctionBodyLengthRule.description,
             location: Location(file: nil, line: 1, character: 1),
             reason: "Function body should span 40 lines or less: currently spans 41 lines")])
+        let longerFunctionBodyWithComments = "func abc() {" +
+            Repeat(count: 40, repeatedValue: " // this is a comment\n").joinWithSeparator("") +
+            "}\n"
+        XCTAssertEqual(violations(longerFunctionBodyWithComments), [])
```

Comment:
> This might be confusing for someone who isn't very familiar with the rule.

I think it would be better `reason` explaining that comment only lines are ignored.

> That tests actually caught a bug

As I tested, SourceKitten returns one token for one multiline comment block. But your code expects that every tokens are in one line. So, counting ignored comment only lines is not correct on multiline comment.